tests/conftest.py

```diff
@@ -1668,7 +1668,7 @@ def check_dut_health_status(duthosts, request):
     If so, we will reload the running config before test case running.
     '''
     check_flag = True
-    if request.config.getoption("enable_macsec"):
+    if request.config.option.enable_macsec:
```

Comment:
What happens if `enable_macsec` not even defined?

Reply:
request.config.option.enable_macsec will return None, rather than raise an error.

Reply:
No, if the option is not defined, it will raise AttributeError. You need to do it this way:
```
if hasattr(request.config.option, 'enable_macsec') and request.config.option.enable_macsec:
```